_includes/share.html

```diff
@@ -1,7 +1,13 @@
-<section class="share">
-  <div class="share-title">Share this article</div>
-  <p>Did this article help you? Share it to inform others.</p>
-  <a class="btn btn-sm twitter" href="https://twitter.com/intent/tweet?text={{ page.title }}&url=https://runnable.com/docs{{ page.url }}" target="_blank">Share to Twitter</a>
-  <a class="btn btn-sm facebook" href="https://www.facebook.com/sharer/sharer.php?u=https://runnable.com/docs{{ page.url }}" target="_blank">Share to Facebook</a>
-  <a class="btn btn-sm linkedin" href="https://www.linkedin.com/shareArticle?mini=true&url=https://runnable.com/docs{{ page.url }}" target="_blank">Share to Linkedin</a>
+<section class="share align-center">
```

Comment:
Prefer if the share class comes last in order, also the naming schema should be `.section-share`